blaze-server/src/main/scala/org/http4s/server/blaze/Http1ServerStage.scala

```diff
@@ -89,9 +98,26 @@ private[blaze] class Http1ServerStage[F[_]](
   // Will act as our loop
   override def stageStartup(): Unit = {
     logger.debug("Starting HTTP pipeline")
+    initIdleTimeout()
     requestLoop()
   }
 
+  private def initIdleTimeout() =
+    idleTimeout match {
+      case f: FiniteDuration =>
+        val cb: Callback[TimeoutException] = {
+          case Left(t) =>
+            fatalError(t, "Error in idle timeout callback")
+          case Right(_) =>
+            logger.debug("Shutting down due to idle timeout")
```

Comment:
We could argue logging level here.  Should perhaps be consistent with the "accepted connection" level.

Reply:
Seems ok.